app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -179,9 +179,10 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
           .isAfter(DateTime.now().toUtc())) {
         continue;
       }
+      final String author = pullRequest['author']['login'];
       final String id = pullRequest['id'];
       final int number = pullRequest['number'];
-      final bool hasApproval =
+      final bool hasApproval = config.rollerAccounts.contains(author) ||
```

Comment:
Can you add a test that checks that an author named `skia-flutter-autoroll-hackerman` does not get merged?

Reply:
This should be covered by other tests, which are testing "some_rando", but I can add this too.